lib/contexts/DiceContext/DiceContext.tsx

```diff
@@ -158,17 +126,22 @@ export function useDice() {
     return { result, playerId: latestPlayerId };
   }
 
+  function clearPool() {
+    setPool([]);
+    setCommandGroups(commandGroupsBeforePool);
+    setCommandGroupsBeforePool([]);
+    setPlayerId(undefined);
+  }
+
```

Comment:
When the pool is rolled, we want to clear it and set the currently selected commands to be the ones that were there before the pool was created